### PR TITLE
Translating Claim Your Data Visualization Certificate to French.

### LINKED ISSUE #7649: French translation of Claim Your Data Visualization Certificate
- [ ] Claim Your Data Visualization Certificate

### PR DESCRIPTION
**Pull request by elazzabi**
Closes #7649

### CODE REVIEW

**Comment by vtamara** [comment]
Good job @elazzabi 
Please ask in the chat channel for (French translation)[ https://gitter.im/FreeCodeCamp/TranslationFrench] that someone reviews your translation.

**Comment by theflametrooper** [comment]
@vtamara This can be merged! No one was available to check the translation so I checked everything in google translate. It all makes sense.

**Comment by BerkeleyTrue** [comment]
@elazzabi Thanks!

File: seed/challenges/02-data-visualization-certification/data-visualization-certificate.json [hunk 1]
```diff
@@ -118,7 +118,34 @@
           ""
         ]
       ],
-      "titleEs": "Reclama tu certificado de Visualización de datos"
+      "titleEs": "Reclama tu certificado de Visualización de datos",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/data-visualization-certificate.json]: @elazzabi you forgot to change title postfix from `Es` to `Fr`.

Comment (elazzabi) [inline seed/challenges/02-data-visualization-certification/data-visualization-certificate.json]: @bugron I think it's just a bug in diff or something. The postfix Fr is in line 148.

Comment (theflametrooper) [inline seed/challenges/02-data-visualization-certification/data-visualization-certificate.json]: Should I submit a pr to revert the pr?

Comment (elazzabi) [inline seed/challenges/02-data-visualization-certification/data-visualization-certificate.json]: @theflametrooper there is no error, everything is fine ;)

Comment (bugron) [inline seed/challenges/02-data-visualization-certification/data-visualization-certificate.json]: @elazzabi ah, yes, you're right, it was false positive. It is so unusual that the title comes after description, that's why I was confused, in full view everything is OK. It is better to keep the title before the description though, like in other files.

**Pull request merged**